client/src/pages/contribute.js

```diff
@@ -37,9 +39,23 @@ const IntakeFormLink = () => (
   />
 )
 
+const InterestFormLink = () => (
+  <Button
+    href={config.links.interest_hsform}
+    label="Complete the Interest Form"
+    margin={{ top: 'small', bottom: 'medium' }}
+    target="_blank"
+    primary
+  />
+)
+
```

Comment:
How about instead of adding this new button as above, we create a generic `FormLinkButton` sub-component? This way, we can define all the necessary button link style rules in one place and share it with both the `IntakeFormLink` and `InterestFormLink` buttons. 

(Comments are just added for aid, except for the one for ESLint `// eslint-disable-...`)

```suggestion
// Generic form link button component responsible for styling
const FormLinkButton = ({ ...props }) => (
  <Button
    margin={{ top: 'small', bottom: 'medium' }}
    target="_blank"
    primary
    // eslint-disable-next-line react/jsx-props-no-spreading
    {...props}
  />
)

const IntakeFormLink = () => (
  <FormLinkButton
    href={config.links.contribute_hsform}
    label="Complete the Intake Form"
  />
)

const InterestFormLink = () => (
  <FormLinkButton
    href={config.links.interest_hsform}
    label="Complete the Interest Form"
  />
)
```

Reply:
Well, if you wanted to do that the better approach would be to create the generic `FormLinkButton` and have that take `href`, and `label` props. Then lower in the file where we map the markdown components to react components, just specify the `href` and `label` props. I think composability is nice, but these are one off components that can have their definition and just map it without props. But then, nothing is stopping you from using a `Button` component and passing all of the props too.

ie if you wanted to map a generic component:

```javascript
    IntakeFormLink: {
      component: FormLinkButton,
      props: {
        href: config.links.contribute_hsform
        label: "Complete the Intake Form"
      }
    },
    InterestFormLink: {
      component: FormLinkButton,
      props: {
        href: config.links.contribute_interest_hsform
        label: "Complete the Interest Form"
      }
    },

```

Reply:
Yes! Defining the `href` and `label` in the `components` prop would be a better approach 👍 !

Reply:
Great ideas! Added. Thanks guys!